libs/dal/src/repositories/subscriber-preference/subscriber-preference.repository.ts

```diff
@@ -6,4 +6,17 @@ export class SubscriberPreferenceRepository extends BaseRepository<SubscriberPre
   constructor() {
     super(SubscriberPreference, SubscriberPreferenceEntity);
   }
+
+  async findAll(
```

Comment:
Should we maybe use a more clear name of what we are finding? But it's not a big deal..

Reply:
@ainouzgali thanks for raising it up.  The method is fetching all the subscriber's temples. What name would you suggest?

Reply:
maybe findSubscriberTemplates? @djabarovgeorge

Reply:
I think it's not fetching templates here but subscriber preference. So `findSubscriberPreferences` would be great. Tend to agree with Gali

Reply:
yea it's the preference, awesome thanks for the feedback! :)